thehive/info.json

```diff
@@ -0,0 +1,2241 @@
+          "type": "text",
+          "description": "Specify the title of the case you want to create in TheHive Platform",
+          "tooltip": "Specify the title of the case you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Description",
+          "name": "description",
+          "type": "text",
+          "description": "Specify the description of the case you want to create in TheHive Platform",
+          "tooltip": "Specify the description of the case you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Summary",
+          "name": "summary",
+          "type": "text",
+          "description": "Specify the summary of the case you want to create in TheHive Platform",
+          "tooltip": "Specify the summary of the case you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Assignee",
+          "name": "assignee",
+          "type": "text",
+          "description": "Specify the assignee of the case you want to create in TheHive Platform",
+          "tooltip": "Specify the assignee of the case you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Severity",
+          "name": "severity",
+          "type": "text",
+          "description": "Specify the severity of the case you want to create in TheHive Platform",
+          "tooltip": "Specify the severity of the case you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Status",
+          "name": "status",
+          "type": "text",
+          "description": "Specify the status of the case you want to create in TheHive Platform",
+          "tooltip": "Specify the status of the case you want to create in TheHive Platform",
+          "visible": true,
+          "editable": true,
+          "required": false
+        },
+        {
+          "title": "Additional Field",
+          "name": "additional_field",
+          "type": "json",
+          "description": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "tooltip": "Specify the additional fields required for the use case. Provide the fields in JSON format.",
+          "visible": true,
+          "editable": true,
+          "required": false
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "title": "Get Case",
+      "operation": "get_case",
+      "description": "Retrieves a specific case information from the TheHive platform based on the input parameters you have specified.",
```

Comment:
Please make the description change: Retrieves a specific case information from the TheHive platform based on the Case ID you have specified.